release-common.sh

```diff
@@ -68,6 +68,45 @@ check_file_transfer_rc () {
     fi
 }
 
+check_for_command () {
+    # check to see if command in $1 is available
+    # returns 0 if command is in path, 1 otherwise
+    command -v $1 2>&1 >/dev/null
```

Comment:
See previous note about `command -v` on bash.

Reply:
Also, you could replace the entire function with
```
check_for_command () {
    type "$1" >/dev/null 2>&1
}
```
because the function returns the exit code of the last statement.

Reply:
Per our discussion, keeping command and  adding a explicit check to see if file is executable.